docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/MapValueToArray.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class MapValueToArray
+    {
+        class DataPoint
+        {
+            public string Timeframe { get; set; }
+        }
+
+        class TransformedData : DataPoint
+        {
+            public int[] Feature { get; set; }
```

Comment:
>Feature [](start = 25, length = 7)

Should be 'Features" as you mentioned this as `Features Column` in the comments below, right?